.github/workflows/releaseTest.yml

```diff
@@ -0,0 +1,17 @@
+name: release-test
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+  release:
+
+jobs:
+  echo-variable:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: See what github.ref is on release
+        run: echo ${{ github.ref }}
```

Comment:
maybe you want `github.ref_name` instead?

Reply:
Yep, I think that will work. I didn't consider it at first because it didn't have "release" in the description 😅